### ISSUE TITLE
KeyError: 'current' in _update_forecast_data line 373

### ISSUE THREAD

**Issue by smurfless1** [IndigoDomotics/rachio-indigo#13]
Symptom: logs full of
```Dec 11, 2022 at 7:25:39 AM
   Rachio Sprinklers Debug         _update_from_rachio
   Rachio Sprinklers Error         Error getting forecast data from Rachio via API.
   Rachio Sprinklers Debug         API error: 
Traceback (most recent call last):
  File "plugin.py", line 373, in _update_forecast_data
    current_conditions = reply_dict["current"]
KeyError: 'current'
```

Other than the obvious key check before entering that logic, what is it used for? Is it ok to just skip that logic?

**Comment by FlyingDiver** [comment]
For some reason, your Rachio account is not returning any of the weather forecast data for your location.  Do you not have the home location set properly in the Rachio mobile app?

**Comment by smurfless1** [comment]
I checked the API by hand, it's just returning the empty dict. Weird.
Address is set in the Account, location is set in the Controller (just re-set it 20 feet to the East just to be sure).

**Comment by FlyingDiver** [comment]
OK, next time I'm working on this code I'll trap that error.

**Comment by smurfless1** [comment]
I actually did that locally, but in case someone else  hits it as well. I don't notice any side effects of having it not do the update. I'm not hurting for this one. Any idea what the gain is for doing that logic that isn't managed by the server?

**Comment by FlyingDiver** [comment]
Not sure I understand the question.  If that data isn't there, you don't get the forecast data in the device custom states.  If you're not using those states, no big deal.  Between this and all the weather plugins out there, it's really redundant data.

**Comment by smurfless1** [comment]
That's all I was curious about. I'm using the NOAA plugin, I'm covered for weather.

**Comment by FlyingDiver** [comment]
Fixed in 2022.0.2
